1.train_models/utils/GreedySubsetSum.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+from PlateTrainingSplits import PlateTrainingSplits
+
+
+class GreedySubsetSum:
+
+    def __init__(self):
+        pass
+
+    def update_test_wells(self, _welldf, _category_col, _well_col, _cell_count_col, _test_well_count):
```

Comment:
Consider adding a function to the docstring

Reply:
I think you mean a docstring to the function. In the case of the init function, I could for completeness, but maybe not needed